src/common/cuepoint/cuepoint-manager.js

```diff
@@ -44,13 +44,16 @@ export class CuePointManager {
   }
 
   getCuePointById(id: string): ?TextTrackCue {
+    let cuePointById = null;
     const metadataTracks = this._getMetadataTracks();
-    metadataTracks.forEach(track => {
+    metadataTracks.every(track => {
```

Comment:
I believe suggestion from Roy clearer or at  least you can use `metadataTracks.some` so on line 54 you can return `cuePoint` without `!`

Reply:
every instead of forEach it's just for breaking the loop once cue point is found, not related to the fix. I prefer it over some since im not looking for a track but over all of them